src/commands/sync/sync.ts

```diff
@@ -55,139 +60,175 @@ async function askForConfirmation(
+        const loadable = await loading(
+          'Deleting unused keys...',
+          opts.client.DELETE('/v2/projects/{projectId}/keys', {
+            params: { path: { projectId: opts.client.getProjectId() } },
+            body: { ids },
+          })
+        );
+
+        handleLoadableError(loadable);
+      }
+    }
 
-  if (opts.removeUnused) {
+    console.log(ansi.bold.green('Sync complete!'));
     console.log(
-      ansi.red(
-        `- ${diff.removed.length} string${diff.removed.length === 1 ? '' : 's'}`
-      )
-    );
-  } else {
-    console.log(
-      ansi.italic(
-        `${diff.removed.length} unused key${
-          diff.removed.length === 1 ? '' : 's'
-        } could be deleted.`
+      ansi.green(
+        `+ ${diff.added.length} string${diff.added.length === 1 ? '' : 's'}`
       )
     );
-  }
 
-  if (opts.backup) {
-    console.log(
-      ansi.blueBright(
-        `A backup of the project prior to the synchronization has been dumped in ${opts.backup}.`
-      )
-    );
-  }
-}
+    if (opts.removeUnused) {
+      console.log(
+        ansi.red(
+          `- ${diff.removed.length} string${
+            diff.removed.length === 1 ? '' : 's'
+          }`
+        )
+      );
+    } else {
+      console.log(
+        ansi.italic(
+          `${diff.removed.length} unused key${
+            diff.removed.length === 1 ? '' : 's'
+          } could be deleted.`
+        )
+      );
+    }
+
+    if (opts.backup) {
+      console.log(
+        ansi.blueBright(
+          `A backup of the project prior to the synchronization has been dumped in ${opts.backup}.`
+        )
+      );
+    }
+  };
 
-export default new Command()
-  .name('sync')
-  .description(
-    'Synchronizes the keys in your code project and in the Tolgee project, by creating missing keys and optionally deleting unused ones. For a dry-run, use `tolgee compare`.'
-  )
-  .addArgument(FILE_PATTERNS)
-  .addOption(EXTRACTOR)
-  .option(
-    '-B, --backup <path>',
-    'Path where a backup should be downloaded before performing the sync. If something goes wrong, the backup can be used to restore the project to its previous state.'
-  )
-  .option(
-    '--continue-on-warning',
-    'Set this flag to continue the sync if warnings are detected during string extraction. By default, as warnings may indicate an invalid extraction, the CLI will abort the sync.'
-  )
-  .option(
-    '-Y, --yes',
-    'Skip prompts and automatically say yes to them. You will not be asked for confirmation before creating/deleting keys.'
-  )
-  .option('--remove-unused', 'Also delete unused keys from the Tolgee project.')
-  .action(syncHandler);
+export default (config: Schema) =>
+  new Command()
+    .name('sync')
+    .description(
+      'Synchronizes the keys in your code project and in the Tolgee project, by creating missing keys and optionally deleting unused ones. For a dry-run, use `tolgee compare`.'
+    )
+    .option(
+      '-B, --backup <path>',
```

Comment:
I am not sure about this, maybe we should tell the users it can only restore keys and translation values, but not states, comments, tags, flags, etc.

Reply:
Yeah, I'm not sure about this either. Can be decieving. However I'll improve the hint.